### PR TITLE
bugfix:ReInject mH in ActivityThread 

### PR DESCRIPTION
**Pull request by xufan**
Yes we inject the `mH` at `onCreate` in `ApkBundleLauncher` but some app like `LBE` could modified it after that stage.
Trace the debug code ,I find out OnePlus2 & MeituV4 was defalut installed app `LBE` in their ROM.And it will inject the `mH` as well after we did.
#412

### CODE REVIEW

**Comment by galenlin** [comment]
函数需要收口下：

```java
    private static void ensureInjectMessageHandler(Object thread) {
        try {
            Field f = thread.getClass().getDeclaredField("mH");
            f.setAccessible(true);
            Handler ah = (Handler) f.get(thread);
            f = Handler.class.getDeclaredField("mCallback");
            f.setAccessible(true);
            
            boolean needsInject = false;
            if (sActivityHandlerCallback == null) {
                needsInject = true;
            } else {
                Object callback = f.get(ah);
                if (callback != sActivityHandlerCallback) {
                    needsInject = true;
                }
            }
            
            if (needsInject) {
                // Inject message handler
                sActivityHandlerCallback = new ActivityThreadHandlerCallback();
                f.set(ah, sActivityHandlerCallback);
            }
        } catch (Exception e) {
            throw new RuntimeException("Failed to replace message handler for thread: " + thread);
        }
    }
```

再统一调用 

```java
ensureInjectMessageHandler(sActivityThread | thread);
```

**Comment by xufan** [comment]
Got it

File: Android/DevSample/small/src/main/java/net/wequick/small/ApkBundleLauncher.java [hunk 1]
```diff
@@ -536,6 +568,7 @@ public void onCreate(Application app) {
         List<ProviderInfo> providers;
         Instrumentation base;
         ApkBundleLauncher.InstrumentationWrapper wrapper;
+        ApkBundleLauncher.ActivityThreadHandlerCallback callback;
```
Comment (galenlin) [inline Android/DevSample/small/src/main/java/net/wequick/small/ApkBundleLauncher.java]: remove this

File: Android/DevSample/small/src/main/java/net/wequick/small/ApkBundleLauncher.java [hunk 2]
```diff
@@ -579,6 +603,7 @@ public void onCreate(Application app) {
         sActivityThread = thread;
         sProviders = providers;
         sHostInstrumentation = base;
+        sActivityThreadHandlerCallback = callback;
```
Comment (galenlin) [inline Android/DevSample/small/src/main/java/net/wequick/small/ApkBundleLauncher.java]: remove this

**Pull request merged**